torch/jit/supported_ops.py

```diff
@@ -40,6 +41,47 @@ def hidden(name):
 
     functions = []
 
+    # Iterate over torch.nn.functional
+    mod = torch.nn.functional
```

Comment:
It's hard for me to review this without seeing its output. Anyway you could post it?